spec/spec.md

```diff
@@ -156,7 +156,7 @@ is outside the scope of this specification. See
 [Presentation Request](#presentation-request).
 
 [[def:Presentation Submission]]
-~ Presentation Submissions are objects embedded within target claim negotiation
+~ Presentation Submissions are objects that ****MAY**** be embedded within target claim negotiation
```

Comment:
Propose reverting this change

Reply:
PE is a data model specification that defines a presentation definition (PD) to describe how a set of credentials may be requested, and a presentation submision (PS) to describe how a set of credentials in a response relate to the presentation definition.
The PE spec cannot mandate how transfer protocols should or should not incorporate PDs and PSs. That is not within its remit. It is up to the other protocol standards bodies to decide if or how to incorporate both PDs and PSs. (They may for example have their own mechanism for either or both.) This is why the above text change has been provided.